src/main/java/ru/abstractmenus/menu/SimpleMenu.java

```diff
@@ -78,7 +78,7 @@ protected void placeItems(Player player) {
                     if (built.getAmount() > 0) {
                         Slot slot = ((InventoryItem) item).getSlot(player, this);
 
-                        if (slot instanceof SlotIndex && ((SlotIndex)slot).getIndex() == -1) {
+                        if (slot instanceof SlotIndex && ((SlotIndex) slot).getIndex() == -1) {
```

Comment:
```suggestion
                        if (slot instanceof SlotIndex slotIndex && slotIndex.getIndex() == -1) {
```

Reply:
There are plenty of moments like this in this class. 
There are also a lot of them by code.

Reply:
????

Reply:
Forget it, I got a little carried away. It doesn't require edits as part of this PR